### ISSUE TITLE
Unable to build frc-docs: permission error

### ISSUE THREAD

**Issue by sciencewhiz** [wpilibsuite/sphinxext-toptranslators#4]
Since sphinxext-toptranslators was added to frc-docs, I'm unable to build frc-docs. 

```
PS C:\Users\Joe\Documents\Robotics\frc-docs> ./make.bat html
Running Sphinx v3.1.1
c:\users\joe\appdata\local\programs\python\python37-32\lib\site-packages\sphinx_tabs\tabs.py:341: RemovedInSphinx40Warning: The app.add_javascript() is deprecated. Please use app.add_js_file() instead.
  app.add_javascript(path)
c:\users\joe\appdata\local\programs\python\python37-32\lib\site-packages\sphinx\deprecation.py:52: RemovedInSphinx40Warning: sphinx.builders.html.DirectoryHTMLBuilder is deprecated. Check CHANGES for Sphinx API modifications.
  return getattr(self._module, name)
hoverxref extension is not fully configured. Tooltips may not work as expected. Check out the documentation for hoverxref_project and hoverxref_version configuration options.
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 342 source files that are out of date
updating environment: [new config] 342 added, 0 changed, 0 removed
reading sources... [  2%] docs/contributing/frc-docs/top-translators
Exception occurred:
  File "c:\users\joe\appdata\local\programs\python\python37-32\lib\shutil.py", line 398, in _rmtree_unsafe
    os.unlink(fullname)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\Joe\\AppData\\Local\\Temp\\tmp4jzst04z\\.git\\objects\\pack\\pack-166749dcf6d7c9bdbeed3c042a811ab2f309cda4.idx'
The full traceback has been saved in C:\Users\Joe\AppData\Local\Temp\sphinx-err-xdfkquf6.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

It says access is denied, however, I'm able to delete that file and directory tree manually.

The log is attached
[sphinx-err-xdfkquf6.log](https://github.com/wpilibsuite/sphinxext-toptranslators/files/4915833/sphinx-err-xdfkquf6.log)

**Comment by TheTripleV** [comment]
Ugh. Windows file permissions. So I added Windows to the tests and they fail on everything except on Python 3.8: https://github.com/TheTripleV/sphinxext-toptranslators/runs/867507938?check_suite_focus=true.

So, when "readonly" are written to disk (git's internal files in this case), they can't be deleted. This was fixed in Python 3.8: https://bugs.python.org/issue26660. I'll try to find a workaround that works for 3.6, 3.7, and pypy3.
